docs/reference-guides/cluster-configuration/rancher-server-configuration/rke1-cluster-configuration.md

```diff
@@ -183,6 +183,19 @@ Clusters that were created before Kubernetes 1.16 will have an `ingress-nginx` `
 
 If the `updateStrategy` of `ingress-nginx` is `OnDelete`, you will need to delete these pods to get the correct version for your deployment.
 
+### Cluster Agent Configuration and Fleet Agent Configuration
+
+You can configure the scheduling fields and resource limits for the Cluster Agent and the cluster's Fleet Agent. You can use these fields to customize tolerations, affinity rules, and resource requirements. Additional tolerations are appended to a list of default tolerations and control plane node taints. If you define custom affinity rules, they override the global default affinity setting. Defining resource requirements sets requests or limits where there previously were none.
+
+:::note
+With this option, it's possible to override or remove rules that are required for the functioning of the cluster. We strongly recommend against removing or overriding the following affinity rules, as this may cause unwanted side effects:
```

Comment:
```suggestion
With this option, it's possible to override or remove rules that are required for the functioning of the cluster. We strongly recommend against removing or overriding the following affinity rules, as this may cause unwanted side effects:
```